docs/operations/metrics.md

```diff
@@ -297,6 +297,8 @@ If the JVM does not support CPU time measurement for the current thread, `ingest
 |`worker/taskSlot/used/count`|Number of busy task slots on the reporting worker per emission period. This metric is only available if the `WorkerTaskCountStatsMonitor` module is included.| `category`, `workerVersion`|Varies|
 |`worker/task/assigned/count`|Number of tasks assigned to an indexer per emission period. This metric is only available if the `WorkerTaskCountStatsMonitor` module is included.|`dataSource`|Varies|
 |`worker/task/completed/count`|Number of tasks completed by an indexer per emission period. This metric is only available if the `WorkerTaskCountStatsMonitor` module is included.|`dataSource`|Varies|
+|`indexer/task/failed/count`|Number of tasks failed on an indexer per emission period. This metric is only available if the `WorkerTaskCountStatsMonitor` module is included.|`dataSource`|Varies|
+|`indexer/task/success/count`|Number of tasks succeeded on an indexer per emission period. This metric is only available if the `WorkerTaskCountStatsMonitor` module is included.|`dataSource`|Varies|
```

Comment:
Why not name these metrics in the same style as `worker/task/completed/count` i.e. `worker/task/success/count` and `worker/task/failed/count`? 

They are being emitted by the same service using the same mechanism. I think the metric names should align to the existing scheme.

Reply:
metric with these names are already being emitted for the MM with different dimensions from overlord-
https://github.com/apache/druid/blob/9dc2569f2276ab01b7917ad968250bd5e20ddfb7/server/src/main/java/org/apache/druid/server/metrics/WorkerTaskCountStatsMonitor.java#L66

Reply:
I think it is okay to emit the metric of the same name from indexer too, since we are already emitting `worker/task/assigned/count`, `worker/task/completed/count` and `worker/task/running/count` from indexers.

Reply:
@kfaraz I've renamed the metrics to the existing names `"worker/task/failed/count"`
and added the dimension in the list of the existing entry in the `defaultMetricsDimensions`